busy_beaver/tasks/github_stats/summary.py

```diff
@@ -15,14 +14,8 @@
 from busy_beaver.models import User
 
 
-def generate_summary(user: User, boundary_dt: datetime):
-    timeline = github.user_activity_after(user.github_username, boundary_dt)
-    user_events = GitHubUserEvents(user, timeline)
-    return user_events.generate_summary_text()
-
-
 class GitHubUserEvents:
-    def __init__(self, user: User, timeline: List[dict]):
+    def __init__(self, user: User, boundary_dt: datetime):
```

Comment:
Cleaned up this module by incorporating the "generate_summary" functionality into the class itself.